src/api/helpers/get-inactive-applications/index.ts

```diff
@@ -0,0 +1,40 @@
+import { Application, Context } from '.keystone/types'; // eslint-disable-line
+import { getThirtyDaysBefore } from '../date';
+import { APPLICATION } from '../../constants';
+
+const { IN_PROGRESS } = APPLICATION.STATUS;
+
+/**
+ * getInactiveApplications
+ * gets inactive applications - not updated for 30 days
+ * returns array of application ids and statuses
+ * @param {Context} context
+ * @returns {Promise<Application[]>} Array of application ids and status
+ */
+const getInactiveApplications = async (context: Context): Promise<Application[]> => {
+  try {
+    console.info('Getting inactive applications - getInactiveApplications helper');
+
+    // timestamp of 30 days ago
+    const thirtyDaysLimit = getThirtyDaysBefore();
```

Comment:
Code is self explanatory, no need for the comment IMO

Reply:
Should probably have a session, maybe this should go in `constants/application`, or the application version definitions - incase it changes

Am also wondering how/if the "30 days" check will work against the existing deadline definition that a user would see, `SUBMISSION_DEADLINE_IN_MONTHS`